app/validation/date_range_check.py

```diff
@@ -0,0 +1,48 @@
+from app.validation.abstract_validator import AbstractValidator
+from app.validation.validation_result import ValidationResult
+from datetime import datetime
+
+DATE_FROM = "From"
```

Comment:
True, but assuming the 1st response is always 'from' and the 2nd 'to' is a more dangerous position, I have to have some kind of handle and at present we don't have one bar the label